compiler/damlc/tests/daml-test-files/Stdlib.daml

```diff
@@ -16,8 +16,6 @@ import DA.List()
 import DA.Logic()
 import DA.Maybe()
 import DA.Monoid()
-import DA.Next.Map()
```

Comment:
is this file useful for anything? if so, why don’t we need those modules anymore?

Reply:
The file describes itself as "Check all the modules in stdlib compile without warnings"

It doesn't make sense to have DA.Next.Map / DA.Next.Set in here anymore since they are supposed to add deprecation warnings. That said this test is not up-to-date. I'd either delete it or update it in a separate PR.